pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: supabase_flutter
 description: Flutter integration for Supabase. This package makes it simple for developers to build secure and scalable products.
-version: 0.2.4
+version: 0.3.0
 homepage: 'https://supabase.io'
 repository: 'https://github.com/supabase/supabase-flutter'
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.13.0 <3.0.0'
   flutter: '>=1.17.0'
```

Comment:
@bdlukaa 
Yeah, this is a weird one. I did some research as well, but couldn't find what exactly gets you the `null-safety` badge. 

I noticed that [hive_flutter](https://pub.dev/packages/hive_flutter) also does not have the `null-safety` badge either even though the [current version](https://pub.dev/packages/hive_flutter/versions) is null safe. 

I was wondering if this line is the cause. We can get rid of this line, if I am correct, right?